lib/screens/v2/candidate_generator/on_bus.ex

```diff
@@ -32,11 +34,18 @@ defmodule Screens.V2.CandidateGenerator.OnBus do
         now \\ DateTime.utc_now(),
         departures_instances_fn \\ &OnBus.Departures.departures_candidates/3
       ) do
-    [fn -> departures_instances_fn.(config, query_params, now) end]
+    [
+      fn -> departures_instances_fn.(config, query_params, now) end,
+      fn -> header_instances(config, now) end
+    ]
     |> Task.async_stream(& &1.())
     |> Enum.flat_map(fn {:ok, instances} -> instances end)
   end
 
+  defp header_instances(config, now) do
+    [%NormalHeader{screen: config, time: now, text: "CONNECTIONS"}]
```

Comment:
I think we can make this a section header of the Departures component, rather than a separate Header component, and that would allow simplifying the frontend somewhat (no new "on-bus header" component required, just styling the markup the Departures component already emits for section headers). In `CandidateGenerator.Widgets.OnBus.Departures`, where we currently have the line `header: %ScreensConfig.Departures.Header{}`, we could specify a `title` field for that struct.

Reply:
Removed the on bus header component and am now passing the `title` field as you suggested! That definitely does simplify things